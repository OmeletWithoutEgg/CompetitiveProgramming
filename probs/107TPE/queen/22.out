-844023.87 19042812.23
