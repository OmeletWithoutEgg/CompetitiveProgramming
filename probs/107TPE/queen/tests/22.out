624964.40 47802819.10
